src/test/resources/network/brightspots/rcv/test_data/2013_minneapolis_mayor/2013_minneapolis_mayor_expected_MINNEAPOLIS_W-1_P-01_precinct_summary.csv

```diff
@@ -13,8 +13,8 @@ Precinct,MINNEAPOLIS W-1 P-01
 Contest Summary
 Number to be Elected,1
 Number of Candidates,36
-Total Number of Ballots,517
-Number of Undervotes (No Rankings),0
```

Comment:
ok looking at the CVR it does look like there is a single undervote which I would expect to be correctly identified now with the fix.
<img width="996" alt="image" src="https://github.com/user-attachments/assets/9310f6fb-f42e-4daf-a328-806907a5144f">
But I'm trying to hunt down why the Total increases by 2. I confirmed in the CVR that there are 519 rows with `W-1 P-01` precinct. 
<img width="1417" alt="image" src="https://github.com/user-attachments/assets/4475fe2c-631c-4d2e-ad1c-2dc3591844c6">

The other precinct file for this contest also increased by two...

Reply:
It looks like:
1. One additional undervote from voter 49004
2. One additional overvote from voter 6938

Reply:
thank you for the point in the right direction. Confirmed that the "Total Number of Ballots" in the slice file is the active ballots in round 1. Since previously we weren't sending inactive ballot counts down to the slices the round 1 count was previously just active ballots. Examples of inactive round 1 ballots are 
- complete undervotes 
- ballots that have only overvotes AND the overvote rule is not so strict that they are inactivated